rules/macos/privilege_escalation_root_crontab_filemod.toml

```diff
@@ -27,7 +27,7 @@ type = "query"
 
 query = '''
 event.category:file and not event.type:deletion and
- file.path:/private/var/at/tabs/root and not process.executable:/usr/bin/crontab
+ file.path:"/private/var/at/tabs/root" and not process.executable:/usr/bin/crontab
```

Comment:
Does this only affect fields related to file? In this case `/usr/bin/crontab` is a path as well, does it need quotes.

Reply:
process executable works well with no quotes as well.